src/resolvers/ens.ts

```diff
@@ -1,21 +1,24 @@
 import axios from 'axios';
 import { StaticJsonRpcProvider } from '@ethersproject/providers';
-import { Contract } from '@ethersproject/contracts';
 import { resize } from '../utils';
 import { max } from '../constants.json';
+import { Contract } from 'ethers';
```

Comment:
Can you revert to @ethersproject/contracts? I'd prefer to install just what's needed

Reply:
reverted